Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -208,6 +198,22 @@ private void OnBackRequested(object sender, BackRequestedEventArgs args)
             }
         }
 
+        protected void UpdateView(bool useTransitions)
+        {
+            string noSelectionState = _stateGroup.CurrentState == _narrowState
+               ? NoSelectionNarrowState
+               : NoSelectionWideState;
+            VisualStateManager.GoToState(this, SelectedItem == null ? noSelectionState : HasSelectionState, useTransitions);
+
+            UpdateViewState();
+
+            // Sets the ability for the Master list to handle focus on interaction. If the details view is the only
+            // view showing, we don't want it to be able to be focused
+            _masterPanel.Visibility = ViewState != MasterDetailsViewState.Details ? Visibility.Visible : Visibility.Collapsed;
```

Comment:
This should probably happen after the animation when going from Master -> Details and before the animation when going from Details -> Master